src/main/java/second_calculator/strCalculator.java

```diff
@@ -0,0 +1,86 @@
+package second_calculator;
+import java.util.Scanner;
+
+public class strCalculator {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("입력 : ");
+
+        String str = sc.nextLine();
+
+        isBlank(str); //빈값여부
+
+        int result = calculator(str);
+
+        System.out.println("최종 값 : " + result);
+    }
+
+
+    public static int calculator(String str) {
+        String[] inputArray = str.split(" "); // 빈값제거
+        int result = Integer.parseInt(inputArray[0]);
+        for (int i = 1; i < inputArray.length; i=i+2) {
+            String symbol = inputArray[i];
+            int val2 = Integer.parseInt(inputArray[i+1]);
+            result = cal(result, symbol, val2);
+        }
+
+        return result;
+    }
+
+    public static int add(int input1, int input2) {
+        return input1 + input2;
+    }
+
+    public static int substract(int input1, int input2) {
+        return input1 - input2;
+    }
+
+    public static int multiply(int input1, int input2) {
+        return input1 * input2;
+    }
+
+    public static int divide(int input1, int input2) {
+        return input1/input2;
```

Comment:
```suggestion
        return input1 / input2;
```
스페이스에 대한 컨벤션을 다른코드와 같이 지켜주는 것은 어떨까요?